### ISSUE TITLE
Settings.throwOnInvalid does not throw for all invalid dates

### ISSUE THREAD

**Issue by JStonevalley** [moment/luxon#494]
Hi
I have found that setting throwOnInvalid does not result in an error being thrown for all dates that are invalid.

The following line results in an invalid date but does not throw:
```javascript
Settings.throwOnInvalid = true  
console.log(DateTime.fromJSDate('2019-04-16T11:32:32Z').isValid) // false
```

Full working example can be found at: https://runkit.com/embed/lr17m8g8q296

**Comment by icambron** [comment]
yup, looks like a bug

**Comment by icambron** [comment]
Fixed in #497
